pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```diff
@@ -122,6 +180,45 @@ func (k *KubeASCheck) parseComponentStatus(sender aggregator.Sender, componentsS
 	}
 	return nil
 }
+
+func (k *KubeASCheck) aggregateEvents(sender aggregator.Sender, events []*v1.Event, modified bool) error {
+	eventsByObject := make(map[string]*kubernetesEventBundle)
+	filteredByType := make(map[string]int)
+
+ITER_EVENTS:
+	for _, event := range events {
+		for _, action := range k.instance.FilteredEventType {
```

Comment:
Could you please add a comment "event type filtering" so the intention is explicit? In python we'd just write `if  action in FilteredEventType`, we loose readability in go :/